cubeviz/controls/slice.py

```diff
@@ -2,7 +2,7 @@
 
 from glue.core import HubListener
 from glue.utils.array import format_minimal
-from specviz.third_party.glue.data_viewer import dispatch as specviz_dispatch
+# from specviz.third_party.glue.data_viewer import dispatch as specviz_dispatch
```

Comment:
There are several places with commented lines that could probably be deleted if they are not going to be used now.